tests/python/train/test_mlp_pruning.py

```diff
@@ -0,0 +1,86 @@
+# pylint: skip-file
+import mxnet as mx
```

Comment:
The code checks whether correct percentages are pruned and correct values are pruned. Do you suggest I test the feature in a different way?